src/components/FrozenFunds/Transfer.jsx

```diff
@@ -32,6 +34,10 @@ class Transfer extends React.Component {
 
 	onDropdownChange(e, value) {
 		this.props.setValue('duration', value);
+		this.setState({duration: value})
```

Comment:
better to be use redux state instead of component state, cause in this project we are prefer working with redux state then component state